ImprovedAnvil/scripts/encounters/spellhold/PPBOOK.baf

```diff
@@ -1,203 +1,219 @@
 IF

```

Comment:
this file is re-formatted with tabs instead of spaces which is not a good practice by itself as well as it makes reading the diff hard because one have to deal with lots of inconsequential changes

Reply:
Yikes.  Any of the changes like that (there are more than a few) are from copying and pasting out of near infinity.  Will reupload

Reply:
thanks